database/index.js

```diff
@@ -2,8 +2,8 @@ const mongoose = require('mongoose');
 mongoose.connect('mongodb://localhost/shirts', { useNewUrlParser: true });
 
 const  shirtSchema = mongoose.Schema({
-  ProductId: Number,
-  ImageURLs: [String]
+  productId: Number,
+  imageURLs: [String]
```

Comment:
nit: Does the scheme type need to be an array?  Should imageURL just be a string?

Reply:
It is an array, because there can be multiple images per product.